unit-tests/Test/Booster/Pattern/Rewrite.hs

```diff
@@ -269,7 +273,7 @@ failsWith t err =
 runRewrite :: Term -> IO (Natural, RewriteResult Term)
 runRewrite t = do
     (counter, _, res) <-
-        runNoLoggingT $ performRewrite False def Nothing Nothing Nothing [] [] $ Pattern_ t
+        runNoLoggingT $ performRewrite (coerce False) def Nothing Nothing Nothing [] [] $ Pattern_ t
```

Comment:
in a similar vein, perhaps these could be made into pattern synonyms, something like
```
pattern CollectRewriteTraces = Flag True
pattern DontCollectRewriteTraces = Flag False
```

Reply:
Good idea! I've done that.